src/sx-tailwind-website/src/pages/_document.js

```diff
@@ -17,20 +17,21 @@ export default class MyDocument extends Document {
 
   render(): Element<'html'> {
     return (
-      <html lang="en-US">
+      <Html lang="en-US">
         <Head>
+          <link rel="icon" href="https://adeira.dev/img/favicon.ico" />
```

Comment:
Would you consider copy-pasting the favicon into this repo? I am personally not a big fan of creating these strange dependencies across projects.